smartsim/_core/_cli/build.py

```diff
@@ -30,62 +30,35 @@
 import typing as t
 from pathlib import Path
 
-import pkg_resources
 from tabulate import tabulate
 
-from smartsim._core._cli.utils import (color_bool, pip_install,
-                                       SMART_LOGGER_FORMAT)
+from smartsim._core._cli.utils import color_bool, SMART_LOGGER_FORMAT
 from smartsim._core._install import builder
 from smartsim._core._install.buildenv import (
     BuildEnv,
     SetupError,
     Version_,
     Versioner,
     DbEngine,
+    VersionConflictError,
```

Comment:
Just double checking, `VersionConflictError` and `SetupError` are both here because they get used inside of `buildenv` too (which we're trying to keep 100% separated from all SmartSim code)?

Reply:
Exactly! The `buildenv` module needs to be imported into the `setup.py` which means it any imports in that module would need to be done through the `importlib` module which can be kind of a pain. As much as I don't love that `VersionConflictError` does not live in the `errors` module, I do think this is the lesser of two evils.